src/libpriv/rpmostree-scripts.cxx

```diff
@@ -442,16 +447,15 @@ run_script_in_bwrap_container (int rootfs_fd,
   if (var_lib_rpm_statedir)
     rpmostree_bwrap_bind_readwrite (bwrap, var_lib_rpm_statedir->path, "/var/lib/rpm-state");
 
-  const gboolean debugging_script = g_strcmp0 (g_getenv ("RPMOSTREE_SCRIPT_DEBUG"), pkg_script) == 0;
+  debugging_script = g_strcmp0 (g_getenv ("RPMOSTREE_SCRIPT_DEBUG"), pkg_script) == 0;
```

Comment:
This one it looks like we can just bring it up to where the var is now declared and that way we keep the `const`.